.github/custard/map.sh

```diff
@@ -0,0 +1,29 @@
+# usage: bash .github/custard/map.sh <script-name> [paths...]
```

Comment:
We might want to make this map.sh and run.sh part of custard at some point. But for quick support they're just bash scripts. It should be easy to port them to custard in the future.